samples/cellular/nrf_cloud_coap_device_message/README.rst

```diff
@@ -0,0 +1,193 @@
+      +CSCON: 1
+      %XTIME: "80","52800241538280","01"
+      +CGEV: ME PDN ACT 0
+      %MDMCH STATUS 2
+      +CEREG: 1,"8169","03117F00",7,0,2,"11100000","11100000"
+      [00:00:05.384,887] <wrn> lte_lc: Registration rejected, EMM cause: 2, Cell ID: 51478272, Tracking area: 33129, LTE mode: 7
+      [00:00:05.400,451] <inf> nrf_cloud_device_message: Connected to LTE
+      [00:00:05.407,257] <inf> nrf_cloud_device_message: Waiting for modem to acquire network time...
+      +CGEV: IPV6 0
+      [00:00:08.417,083] <inf> nrf_cloud_device_message: Network time obtained
+      [00:00:08.424,224] <inf> nrf_cloud_info: Device ID: 7e699894-79b6-11f0-a2b4-db93a314a2aa
+      [00:00:08.433,288] <inf> nrf_cloud_info: IMEI:      359400123456789
+      [00:00:08.558,837] <inf> nrf_cloud_info: UUID:      7e699894-79b6-11f0-a2b4-db93a314a2aa
+      [00:00:08.567,932] <inf> nrf_cloud_info: Modem FW:  mfw_nrf91x1_2.0.2
+      [00:00:08.574,829] <inf> nrf_cloud_info: Protocol:          CoAP
+      [00:00:08.581,298] <inf> nrf_cloud_info: Download protocol: HTTPS
+      [00:00:08.587,860] <inf> nrf_cloud_info: Sec tag:           16842753
+      [00:00:08.594,696] <inf> nrf_cloud_info: CoAP JWT Sec tag:  16842753
+      [00:00:08.601,531] <inf> nrf_cloud_info: Host name:         coap.nrfcloud.com
+      [00:00:10.231,933] <inf> nrf_cloud_coap_transport: Request authorization with JWT
+      [00:00:10.495,574] <inf> nrf_cloud_coap_transport: Authorization result_code: 2.01
+      [00:00:10.503,692] <inf> nrf_cloud_coap_transport: Authorized
+      [00:00:10.510,101] <inf> nrf_cloud_coap_transport: DTLS CID is active
+      [00:00:10.970,703] <inf> nrf_cloud_device_message: Checking for shadow delta...
+      [00:00:11.449,615] <inf> nrf_cloud_device_message: Initial config sent
+      [00:00:11.456,573] <inf> nrf_cloud_device_message: Reset reason: 0x10000
+      [00:00:11.684,783] <inf> nrf_cloud_device_message: Sending message:'{"sample_message":"Hello World, from the CoAP Device Message Sample! Message ID: 1755700534318"}'
+      [00:00:11.928,649] <inf> nrf_cloud_device_message: Message sent
+      [00:00:12.035,125] <inf> nrf_cloud_device_message: Sent Hello World message with ID: 1755700534318
+      [00:00:16.847,137] <inf> nrf_cloud_device_message: Sending message:'{"appId":"BUTTON", "messageType":"DATA", "data":"1"}'
+      [00:00:17.136,352] <inf> nrf_cloud_device_message: Message sent
+      [00:00:17.242,858] <inf> nrf_cloud_device_message: Checking for shadow delta...
+      [00:00:17.774,841] <inf> nrf_cloud_device_message: Initial config sent
+      +CSCON: 0
+
+Troubleshooting
+===============
+
+If you are not getting the output similar to the one in `Testing`_, check the following potential issues:
+
+The network carrier does not provide date and time
+   The sample requires the network carrier to provide date and time to the modem.
+   Without a valid date and time, the modem cannot generate JWTs with an expiration time.
```

Comment:
```suggestion
If you are not getting the output similar to the one in `Testing`_, check the network carrier, which might not provide a date and time. 
The sample requires the network carrier to provide a date and time to the modem. 
Without a valid date and time, the modem cannot generate JWTs with an expiration time.
```
Was it intentional to have this sentence and the details in the definition list format? We could go with above as this is just a single point?